code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -101,12 +101,9 @@
 /obj/machinery/atmospherics/Initialize(mapload)
 	if(mapload && name != initial(name))
 		override_naming = TRUE
-	var/turf/turf_loc = null
-	if(isturf(loc))
-		turf_loc = loc
-		turf_loc.add_blueprints_preround(src)
```

Comment:
that or make a wrapper that does this. idk. honestly not really sure why this is a proc on the turf could just as easily be on movable

Reply:
I don't really like the duplication of this either but I really don't like solving it with a macro like this